packages/fx-core/src/common/azure-hosting/utils.ts

```diff
@@ -49,25 +54,26 @@ async function getAzureAccountCredential(
 }
 
 async function fetchWebSiteManagementClient(
-  tokenProvider: TokenProvider,
-  inputs: Inputs
+  subscriptionId: string,
+  tokenProvider: TokenProvider
 ): Promise<appService.WebSiteManagementClient> {
   return new appService.WebSiteManagementClient(
     await getAzureAccountCredential(tokenProvider),
-    inputs.subscriptionId
```

Comment:
`inputs` may not contain subscription Id.
We should read this kind of information from envInfo.